src/core/api_client/tests/factories/auth.factory.ts

```
@@ -0,0 +1,34 @@
+    return `${baseUrl}/realms/${realm}/protocol/openid-connect/token`;
+  }
+
+  static getLogoutEndpoint(config: AuthConfig = {}): string {
+    const { baseUrl, realm } = this.createConfig(config);
+    return `${baseUrl}/realms/${realm}/protocol/openid-connect/logout`;
+  }
+
+  static getApiUrl(path: string = ''): string {
+    return `http://localhost:8080/api${path}`;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)